cypress/support/beforeEach.js

```diff
@@ -0,0 +1,7 @@
+beforeEach(function() {
+  // Reset DB before each test
+  // TODO: find a faster way to do this.
+  cy.exec(
+    "docker-compose run web sh -c './manage.py flush --noinput && ./manage.py generate_test_data'"
+  );
```

Comment:
Do you think we could do something like? 
```
if (Cypress.env('docker')) {
  cy.exec(
    "docker-compose run web sh -c './manage.py flush --noinput && ./manage.py generate_test_data'"
  );
} else {
  cy.exec(
    "python manage.py flush && python manage.py generate_test_data"
  )
}

Reply:
Yeah, we should. I was trying to figure out if there was a clever way to make it work without an envvar. Hmmmmm.

Reply:
Either way, not a blocker for merging I think... getting the new production stuff shipped is priority right now.

Reply:
You're right.